### PR TITLE
Update for siproxd-0.8.2

### PR DESCRIPTION
**Pull request by hb9xar**
These changes implement the changes needed to update siproxd to the current 0.8.2 version.

Changes in net/pfSense-pkg-siproxd:
- updated the config page to support most available settings
- added config support for all new plugins
- updated the show "Registered Phones" page to correctly display the registered UAs (slight change of storage format)

Changes in net/siproxd:
- adapted Makefile, distinfo and pkg-plist according to current FreeBSD ports tree

### CODE REVIEW

**Comment by pfSense-git-updates** [comment]
Before this pull request can be accepted you must first sign a CLA as described at https://www.pfsense.org/about-pfsense/#cla. Please read for more details.

File: net/pfSense-pkg-siproxd/Makefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # $FreeBSD$
 
 PORTNAME=	pfSense-pkg-siproxd
-PORTVERSION=	1.1.2
-PORTREVISION=	2
+PORTVERSION=	1.1.3
+PORTREVISION=	1
```
Comment (rbgarga) [inline net/pfSense-pkg-siproxd/Makefile]: PORTREVISION line should be removed when PORTVERSION is increased

**Comment by rbgarga** [comment]
Please remove changes made to net/siproxd since it's already in 0.8.2 on pfSense/FreeBSD-ports

**Comment by hb9xar** [comment]
- updated net/pfSense-pkg-siproxd/Makefile (removed PORTREVISION)
- Reverted net/siproxd/ changes

**Comment by hb9xar** [comment]
Any progress? 
Please advice if further changes are required.

**Comment by rbgarga** [comment]
I'm merging it to 2.4.0 and 2.3.3 snapshots so more people can test the changes. After it's well tested I'll cherry-pick to 2.3.2-RELEASE

**Pull request merged**